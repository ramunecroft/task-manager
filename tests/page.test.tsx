import "@testing-library/jest-dom";
import TaskPage from "@/app/task/page";
import {render} from "@testing-library/react";

describe("Page", () => {
  it("renders a heading", () => {
    const {container} = render(<TaskPage />);
    expect(container).toMatchSnapshot();
  });
});
