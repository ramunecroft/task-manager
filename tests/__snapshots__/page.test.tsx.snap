// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders a heading 1`] = `<div />`;
